pupil_src/launchables/player.py

```diff
@@ -531,7 +531,6 @@ def handle_notifications(n):
             p.alive = False
         g_pool.plugins.clean()
 
-        g_pool.capture.cleanup()
```

Comment:
Did we call `g_pool.capture.cleanup()` twice by accident in the recent releases?